tests/user.spec.ts

```diff
@@ -9,7 +9,7 @@ test.describe('/api/users', () => {
         expect(user.username).toBe(config.user.username);
         expect(user.email).toBe(config.user.email);
         expect(user.confirmed).toBe(config.user.confirmed);
-        expect(user.projects.length).toBe(1);
+        expect(user.projects.length).not.toBe(0);
         const project = user.projects[0];
```

Comment:
Also another question.
In this scenario is it possible the user have no projects ?
Because if it is possible and we check the length of projects (and projects is nulll) it could return an error.

Reply:
Hi @tiagopazhs Thank you pointing out. Yes you are right, in real world it may happen. But in our test env this will not be the case anytime. If you see in common.js file we are creating project as part of setup and projects are assigned to the config.user. Also in user.spec.ts we are using the same user from config.

Reply:
Yes, Also user can have more than one project. That case we have covered as part of test setup process.